eng/Packages.Data.props

```diff
@@ -47,8 +47,8 @@
     <PackageReference Update="System.Security.Cryptography.Algorithms" Version="4.3.0" />
     <PackageReference Update="System.Security.Cryptography.Cng" Version="4.3.0" />
     <PackageReference Update="System.Security.Cryptography.Primitives" Version="4.3.0" />
-    <PackageReference Update="System.Text.Encodings.Web" Version="4.5.1" />
-    <PackageReference Update="System.Text.Json" Version="4.7.2" />
+    <PackageReference Update="System.Text.Encodings.Web" Version="6.0.0" />
```

Comment:
Let's not bump dependencies for the T1 libraries.

Reply:
reverted